www/public/repl/repl.js

```diff
@@ -32,7 +31,7 @@ const repl = {
   app: null,
 
   // Temporary storage for values passing back and forth between JS and Wasm
-  result: { addr: 0, buffer: new ArrayBuffer() },
+  result_addr: { addr: 0, buffer: new ArrayBuffer() },
```

Comment:
Oh wow, this is wrong. This field was changed to a number everywhere else.
The initial value always get overwritten, so everything works. But it's confusing.
I'll make another PR.